TRIAGE_GUIDE.md

```diff
@@ -0,0 +1,91 @@
+# # O3DE Network SIG - Issue Triage Guide
+
+# Overview
+This guide covers how to triage GitHub issues for SIG-Network. Maintainers are encouraged to use and update this guide to ensure
+any contributor to SIG-Network understands how issues are handled.
+
+
+##  Issue Triaging
+Triaging is the process used to handle intake of issues into the SIG-network backlog. The process aims to ensure issues are both relevant to SIG-Network
+and contain sufficient information so that the community can take action.
+
+Process aims to ensure:
+* The issue is appropriate for SIG-Network.
+* The issue has clear information as to the nature of the issue.
+* Issues are regularly maintained and updated until they are resolved.  
+* That reported issues are an actual issues, rather than a request for help or an issue for another SIG.
+
+
+# Process
+SIG-Network will triage issues once a week on [Thursdays](https://lists.o3de.org/g/o3de-calendar/viewevent?repeatid=39342&eventid=1263668&calstart=2022-01-20). Anyone is welcome to attend. Triage will be led by SIG chair or maintainer.
+
+Triaging aims to:
+* Ensure issues in backlog are in a ready state for the community to take action upon.
+* Ensure load is balanced across SIG maintainers and participants.
+* Involve the SIG-Network community so all can participate.
+
+If time permits, on the day of triage and before the meeting, create a new thread in SIG-Network and add triage links below.
+* Set the thread to automatically archive after 24 hours.
+
+## Triage Links
+* Main O3DE repository: https://github.com/o3de/o3de/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-triage+label%3Asig%2Fnetwork 
+* Multiplayer Sample: https://github.com/o3de/o3de-multiplayersample/issues
+* NetSoak Test: https://github.com/o3de/o3de-netsoaktest/issues
+
+## Brief Overview
+1. Join the SIG-Network discord voice channel
+2. Start with main repro link
+3. Process all new main repository issues. New issues in main repository should have labels `needs-triage` and `sig/network`
```

Comment:
I understand adding/removing labels requires maintainer(?*) level permissions. It may be good to add a "Who can triage?" section or bullet to the overview. 

*(likely a bigger discussion than just for this SIG) Do we have ideas re: how to allow non-maintainers to assist with traige? E.g., a bot to invoke to add labels, or intermediate permission levels which can edit labels?

Reply:
All SIG(s) should have maintainers and reviewers that are active. Someone has to empowered to act on behalf of the SIG to accept issues. If we had SIGs without maintainers that would be a concern. All SIGs have the power to promote maintainers, so it would be interesting to understand the reason. But its probably more a question for SIG-Ops/SIG-Build (as they manage GitHub permissions).

Reply:
Generally, I think there's value in providing avenues for a broader set of contributors to assist with triage, both from a participation and load balancing of work across the community perspectives, but agreed that this is a discussion better had with SIG-Ops/SIG-Build.